### PR TITLE
[IMP] db_management: rework section for future content

### PR DESCRIPTION
**Pull request by dbo-odoo**
The "database management" section was just a big fourre-tout.

This commit splits a new "Practical Info" section from the
"Applications" section and split the db management page in
3 sub-pages that will be soon extended.

### CODE REVIEW

File: applications.rst [hunk 1]
```diff
@@ -18,8 +18,6 @@ Applications
    project
    expense/expense
    general
-   getting_started/documentation
-   db_management/documentation
    legal
```
Comment (dbo-odoo) [inline applications.rst]: not sure: maybe legal should be moved to this new section as well

Comment (tivisse) [inline applications.rst]: For sure Damien

**Comment by tivisse** [comment]
Landed in 10.0 at https://github.com/odoo/documentation-user/commit/60ab9a5634ce65104998b0bb8b42b1fc2607d69e

`Legal` page also moved to `Practical Information` at https://github.com/odoo/documentation-user/commit/d41451129b94ef8dccc072c104f812d0728ab00d

Thank @dbo-odoo, have a nice day.

**Comment by dbo-odoo** [comment]
:turkey:

**Pull request closed without merging**